gammapy/irf/effective_area.py

```diff
@@ -252,10 +252,14 @@ def from_parametrization(cls, energy_axis_true=None, instrument="HESS"):
         pars = {
             "HESS": [6.85e9, 0.0891, 5e5],
             "HESS2": [2.05e9, 0.0891, 1e5],
-            "CTA": [1.71e11, 0.0891, 1e5],
+            "CTAO": [1.71e11, 0.0891, 1e5],
         }
 
-        if instrument not in pars.keys():
+        if instrument == "CTA":
+            ss = "Deprecation Warning> "
+            ss += "since v1.3, the value 'CTA' is replaced by 'CTAO' for the argument instrument."
+            raise ValueError(ss)
+        elif instrument not in pars.keys():
```

Comment:
```suggestion
        if instrument == "CTA":
            instrument = "CTAO"
            warnings.warn(
                "Since v1.3, the value 'CTA' is replaced by 'CTAO' for the argument instrument.",
                GammapyDeprecationWarning,
            )
        if instrument not in pars.keys():
```